### ISSUE TITLE
crashes with C9 or Cadd9 or C add9 or C add 9 etc.

### ISSUE THREAD

**Issue by chillin-** [pedrozath/coltrane#62]
Welcome to Coltrane 4.1.1 (You chose: chords)
? What do you need (You chose: show chord)
? Which chord?                                                                                                   
> C add 9                                                                                                        
? How do you wanna see it? (You chose: Guitar)
/opt/local/lib/ruby2.7/gems/2.7.0/gems/coltrane-4.1.1/lib/coltrane/theory/note.rb:29:in `initialize'             
/opt/local/lib/ruby2.7/gems/2.7.0/gems/coltrane-4.1.1/lib/coltrane/theory/note.rb:43:in `new'                    
/opt/local/lib/ruby2.7/gems/2.7.0/gems/coltrane-4.1.1/lib/coltrane/theory/note.rb:43:in `[]'                     
/opt/local/lib/ruby2.7/gems/2.7.0/gems/coltrane-4.1.1/lib/coltrane/theory/chord.rb:93:in `parse_from_name'       
/opt/local/lib/ruby2.7/gems/2.7.0/gems/coltrane-4.1.1/lib/coltrane/theory/chord.rb:22:in `initialize'            
/opt/local/lib/ruby2.7/gems/2.7.0/gems/coltrane-4.1.1/lib/coltrane/commands/get_chords_from_string.rb:6:in `new' 
/opt/local/lib/ruby2.7/gems/2.7.0/gems/coltrane-4.1.1/lib/coltrane/commands/get_chords_from_string.rb:6:in `block in run'
/opt/local/lib/ruby2.7/gems/2.7.0/gems/coltrane-4.1.1/lib/coltrane/commands/get_chords_from_string.rb:5:in `map'
/opt/local/lib/ruby2.7/gems/2.7.0/gems/coltrane-4.1.1/lib/coltrane/commands/get_chords_from_string.rb:5:in `run'
/opt/local/lib/ruby2.7/gems/2.7.0/gems/coltrane-4.1.1/lib/coltrane/commands/command.rb:7:in `run'
/opt/local/lib/ruby2.7/gems/2.7.0/gems/coltrane-4.1.1/lib/coltrane/ui/views/show_chord.rb:14:in `render'
/opt/local/lib/ruby2.7/gems/2.7.0/gems/coltrane-4.1.1/lib/coltrane/ui/base_view.rb:5:in `render'
/opt/local/lib/ruby2.7/gems/2.7.0/gems/gambiarra-0.0.5/lib/gambiarra/view.rb:34:in `respond'
/opt/local/lib/ruby2.7/gems/2.7.0/gems/gambiarra-0.0.5/lib/gambiarra/route.rb:13:in `respond'
/opt/local/lib/ruby2.7/gems/2.7.0/gems/gambiarra-0.0.5/lib/gambiarra/history.rb:31:in `refresh'
/opt/local/lib/ruby2.7/gems/2.7.0/gems/gambiarra-0.0.5/lib/gambiarra/router.rb:21:in `get'
/opt/local/lib/ruby2.7/gems/2.7.0/gems/gambiarra-0.0.5/lib/gambiarra/app.rb:82:in `flow'
/opt/local/lib/ruby2.7/gems/2.7.0/gems/gambiarra-0.0.5/lib/gambiarra/app.rb:26:in `block in start'
/opt/local/lib/ruby2.7/gems/2.7.0/gems/gambiarra-0.0.5/lib/gambiarra/app.rb:26:in `loop'
/opt/local/lib/ruby2.7/gems/2.7.0/gems/gambiarra-0.0.5/lib/gambiarra/app.rb:26:in `start'
/opt/local/lib/ruby2.7/gems/2.7.0/gems/coltrane-4.1.1/exe/coltrane:24:in `<top (required)>'
/opt/local/bin/coltrane:23:in `load'
/opt/local/bin/coltrane:23:in `<main>'
Bad constructor. Wrong argument(s).
Traceback (most recent call last):
        5: from /opt/local/bin/coltrane:23:in `<main>'
        4: from /opt/local/bin/coltrane:23:in `load'
        3: from /opt/local/lib/ruby2.7/gems/2.7.0/gems/coltrane-4.1.1/exe/coltrane:24:in `<top (required)>'
        2: from /opt/local/lib/ruby2.7/gems/2.7.0/gems/gambiarra-0.0.5/lib/gambiarra/app.rb:26:in `start'
        1: from /opt/local/lib/ruby2.7/gems/2.7.0/gems/gambiarra-0.0.5/lib/gambiarra/app.rb:26:in `loop'
/opt/local/lib/ruby2.7/gems/2.7.0/gems/gambiarra-0.0.5/lib/gambiarra/app.rb:26:in `block in start': no implicit conversion of nil into Hash (TypeError)

**Comment by martinmr** [comment]
I played around for a bit and all the issues seem to be related to the gambiarra gem, which is authored by the author of this gem and has not been updated since around the same time. So probably it's just some issue related to a new version of ruby or some other library that is causing it to break.

**Comment by kianmeng** [comment]
I've also encountered similar issue:

```
$ coltrane                                                                                   
? Welcome to Coltrane 4.1.1 (You chose: chords)                                                         
? What do you need (You chose: show chord)                                                              
? Which chord?                                                                                          
[0;94m> [0m[33m                                                                                         
? How do you wanna see it? (You chose: Guitar)                                                          
/var/lib/gems/3.0.0/gems/coltrane-4.1.1/lib/coltrane/commands/get_representation_chords.rb:12:in `block 
in run'                   
/var/lib/gems/3.0.0/gems/coltrane-4.1.1/lib/coltrane/commands/get_representation_chords.rb:5:in `map'   
/var/lib/gems/3.0.0/gems/coltrane-4.1.1/lib/coltrane/commands/get_representation_chords.rb:5:in `run'   
/var/lib/gems/3.0.0/gems/coltrane-4.1.1/lib/coltrane/commands/command.rb:7:in `run'                     
/var/lib/gems/3.0.0/gems/coltrane-4.1.1/lib/coltrane/ui/views/show_chord.rb:15:in `render'              
/var/lib/gems/3.0.0/gems/coltrane-4.1.1/lib/coltrane/ui/base_view.rb:5:in `render'
/var/lib/gems/3.0.0/gems/gambiarra-0.0.5/lib/gambiarra/view.rb:34:in `respond'
/var/lib/gems/3.0.0/gems/gambiarra-0.0.5/lib/gambiarra/route.rb:13:in `respond'
/var/lib/gems/3.0.0/gems/gambiarra-0.0.5/lib/gambiarra/history.rb:31:in `refresh'
/var/lib/gems/3.0.0/gems/gambiarra-0.0.5/lib/gambiarra/router.rb:21:in `get'
/var/lib/gems/3.0.0/gems/gambiarra-0.0.5/lib/gambiarra/app.rb:82:in `flow'
/var/lib/gems/3.0.0/gems/gambiarra-0.0.5/lib/gambiarra/app.rb:26:in `block in start'
/var/lib/gems/3.0.0/gems/gambiarra-0.0.5/lib/gambiarra/app.rb:26:in `loop'
/var/lib/gems/3.0.0/gems/gambiarra-0.0.5/lib/gambiarra/app.rb:26:in `start'
/var/lib/gems/3.0.0/gems/coltrane-4.1.1/exe/coltrane:24:in `<top (required)>'
/usr/local/bin/coltrane:25:in `load'
/usr/local/bin/coltrane:25:in `<main>'
undefined method `name' for nil:NilClass
/var/lib/gems/3.0.0/gems/gambiarra-0.0.5/lib/gambiarra/app.rb:26:in `block in start': no implicit conver
sion of nil into Hash (TypeError)
        from /var/lib/gems/3.0.0/gems/gambiarra-0.0.5/lib/gambiarra/app.rb:26:in `loop'
        from /var/lib/gems/3.0.0/gems/gambiarra-0.0.5/lib/gambiarra/app.rb:26:in `start'
        from /var/lib/gems/3.0.0/gems/coltrane-4.1.1/exe/coltrane:24:in `<top (required)>'
        from /usr/local/bin/coltrane:25:in `load'
        from /usr/local/bin/coltrane:25:in `<main>
```

**Comment by pedrozath** [comment]
definitely not being able to find a chord shouldn't crash the whole app. will investigate this during the weekend

**Comment by pedrozath** [comment]
fixed by #68

**Comment by chillin-** [comment]
Welcome to Coltrane 4.1.2 (You chose: chords)
? What do you need (You chose: show chord)
? Which chord?                                                                                                   
> Cadd9                                                                                                          
? How do you wanna see it? (You chose: Guitar)
**The chord you provided wasn't found.** If you're sure this chord exists, would you mind to suggest it's inclusion here: https://github.com/pedrozath/coltrane/issues                                                                
                                                                                                                 

Works perfectly now. Thanks.
